### PR TITLE
Fixes #776] Include the upload limits inside __GEONODE_CONFIG__

### LINKED ISSUE #776: Include the upload limits inside __GEONODE_CONFIG__
We already have the `api/v2/upload-size-limits` endpoint but to avoid an extra request we want to set the current `dataset_upload_size` and `document_upload_size` values inside [__GEONODE_CONFIG__](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html#L60):

```javascript
"__GEONODE_CONFIG__ ": {
    ...
    "localSettings": {
        ...,
        "geoNodeSettings": {
            ...,
            "documentMaxUploadSize": <upload size in bytes>,
            "datasetMaxUploadSize": <upload size in bytes>,
            ...
        },
        ...
    }
}
```

The values will be obtained from the UploadSize model using the specific slugs-
E.g. `UploadSizeLimit.objects.get(slug="dataset_upload_size")`

### PR DESCRIPTION
**Pull request by marthamareal**
References #776

### CODE REVIEW

File: geonode_mapstore_client/context_processors.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def resource_urls(request):
         'DEFAULT_MAP_CRS': getattr(settings, "DEFAULT_MAP_CRS", 'EPSG:3857'),
         'DEFAULT_MAP_ZOOM': getattr(settings, "DEFAULT_MAP_ZOOM", 0),
         'DEFAULT_TILE_SIZE': getattr(settings, "DEFAULT_TILE_SIZE", 512),
+        'DEFAULT_MAX_UPLOAD_SIZE': getattr(settings, "DEFAULT_MAX_UPLOAD_SIZE", 104857600),
```
Comment (allyoucanmap) [inline geonode_mapstore_client/context_processors.py:29]: @marthamareal @giohappy 
Should we gat the upload size from the db instead? and should we have two different variables?

File: geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html [hunk 2]
```diff
@@ -88,7 +89,9 @@
                     defaultLayerFormat: defaultLayerFormat,
                     timeEnabled: timeEnabled,
                     allowedDocumentTypes: allowedDocumentTypes,
-                    isMobile: isMobile
+                    isMobile: isMobile,
+                    dataset_upload_size: defaultMaxUploadSize,
+                    document_upload_size: defaultMaxUploadSize,
```
Comment (allyoucanmap) [inline geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html:94]: please use camelCase for variables in js
```suggestion
                    datasetMaxUploadSize: defaultMaxUploadSize,
                    documentMaxUploadSize: defaultMaxUploadSize,
```

**Review by giohappy — CHANGES_REQUESTED** [review]
@marthamareal please read the issue carefully. We need to obtain the dataset and document sizes from the DB.
I've also updated it with the key in camelCase form.

File: geonode_mapstore_client/context_processors.py (hunk 1 above)
Comment (marthamareal) [reply geonode_mapstore_client/context_processors.py]: updated

File: geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html (hunk 2 above)
Comment (marthamareal) [reply geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html]: updated

File: geonode_mapstore_client/context_processors.py [hunk 3]
```diff
@@ -10,6 +10,18 @@
 #########################################################################
 
 from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+
+from geonode.upload.models import UploadSizeLimit
+
+
+def get_max_upload_size(slug):
```
Comment (giohappy) [inline geonode_mapstore_client/context_processors.py:18]: @marthamareal I would put this under the upload [utils.py](https://github.com/GeoNode/geonode/blob/5b1e7ae6b74658a63090a7dc68c60996542e8c8d/geonode/upload/utils.py) module

Comment (giohappy) [reply geonode_mapstore_client/context_processors.py]: ok, waiting for the PR on geonode

**Review by giohappy — APPROVED** [review]

**Pull request merged**